src/app/public/modules/demo/demo-control-panel.component.html

```diff
@@ -14,5 +14,14 @@
         {{ 'sky-docs-demo-control-panel-reset-button-label' | skyLibResources }}
       </button>
     </div>
+    <ng-container *ngIf="demoThemeSvc.supportsTheming | async">
+      <sky-restricted-view>
+        <div
+          class="sky-docs-demo-control-panel-column sky-docs-demo-control-panel-theme-column"
+        >
+          <sky-docs-demo-control-panel-theme (themeSettingsChange)="themeSettingsChange($event)"></sky-docs-demo-control-panel-theme>
```

Comment:
Code style, attributes on separate lines.

```suggestion
          <sky-docs-demo-control-panel-theme
            (themeSettingsChange)="themeSettingsChange($event)"
          ></sky-docs-demo-control-panel-theme>
```